packages/babel-traverse/src/path/evaluation.js

```diff
@@ -112,25 +112,19 @@ export function evaluate(): { confident: boolean; value: any } {
     }
 
     if (path.isTemplateLiteral()) {
-      let str = "";
-
-      let i = 0;
-      const exprs = path.get("expressions");
-
-      for (const elem of (node.quasis: Array<Object>)) {
-        // not confident, evaluated an expression we don't like
-        if (!confident) break;
+      return evaluateQuasis(path, node.quasis);
+    }
 
-        // add on cooked element
-        str += elem.value.cooked;
+    if (path.isTaggedTemplateExpression()) {
```

Comment:
Thanks, fixed in https://github.com/babel/babel/pull/5681/commits/af126dfc45c8d2bf11f353ef53b776f807ed4c66. I'm curious, though, is this check made redundant by the checks [here](https://github.com/babel/babel/pull/5681/files#diff-3527506de741b25a9b8d0e9fcabf11a2R123)? I'm not very familiar with Babel internals yet.